examples/apply-replacements-simple/README.md

```diff
@@ -0,0 +1,65 @@
+# apply-replacements: Simple Example
+
+### Overview
+
+In this example, we will see how to invoke the apply-replacements function
+with a simple example
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/apply-replacements-simple
+```
+
+We use a `ApplyReplacements` object to configure the `apply-replacements` function. 
+
+```yaml
+apiVersion: fn.kpt.dev/v1alpha1
+kind: ApplyReplacements
```

Comment:
Should this be singular `ApplyReplacement` ?  It's a config kind that can contain replacements.

Reply:
Hm, we can. I don't have a strong opinion, but I am slightly in favor of plural `ApplyReplacements` because (1) the function name is apply-replacements which is plural and (2) because this one function can apply multiple replacements. `ApplyReplacement` might imply that the function can only apply one replacement per invocation.  

However I am ok to change it if you feel strongly.